src/flash/flashTask.ts

```diff
@@ -138,6 +148,15 @@ export class FlashTask {
     return new vscode.ProcessExecution(pythonBinPath, flasherArgs, options);
   }
 
+  public _dfuFlashing() {
+    this.flashing(true);
+    return new vscode.ShellExecution(
```

Comment:
This method of flashing doesn't consider the case where multiple chips are present and how to choose between devices. As shown in [DFU documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32s2/api-guides/dfu.html), you should use the `--path` argument and provide the user with the chance to choose which device to use.